CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ We'd love for you to contribute to our source code and to make Recharts even bet
 
 ## Ongoing Initiatives
 
-We use Github Discussion to organise our discussion along new intiatives and organise ourselves. Please do check the [announcements](https://github.com/recharts/recharts/discussions/categories/announcements) section for an overview of recent initiatives.
+We use Github Discussion to organize our discussion along new initiatives and organize ourselves. Please do check the [announcements](https://github.com/recharts/recharts/discussions/categories/announcements) section for an overview of recent initiatives.
```

Comment:
`organise` is the correct spelling here for UK english, but all good

Reply:
should I get it corrected @ckifer or is it good to merge?

Reply:
Nope its good, I was just waiting on the build